From: Roland Rosenfeld <roland@debian.org>
Subject: Extend xfig.desktop with some additional information.

--- a/xfig.desktop
+++ b/xfig.desktop
@@ -1,7 +1,11 @@
-[Desktop Entry]
-Exec=/usr/X11R6/bin/xfig
-InitialPreference=2
-MimeType=image/x-xfig;
-Name=xfig
-Terminal=false
-Type=Application
+[Desktop Entry]
+Exec=/usr/bin/xfig %f
+InitialPreference=2
+MimeType=image/x-xfig;
+Name=xfig
+Comment=Facility for Interactive Generation of figures under X11
+Icon=xfig
+Terminal=false
+Type=Application
+Categories=Graphics;
+Keywords=vector;graphics;figure;drawing
